### PR TITLE
replace caddy to nginx for network part

### PR DESCRIPTION
**Pull request by zhaozhanqi**
@lihongan  ^^

### CODE REVIEW

File: features/networking/egress-ip.feature [hunk 1]
```diff
@@ -354,11 +354,12 @@ Feature: Egress IP related features
     """
 
     # Create a pod
-    Given I obtain test data file "routing/caddy-docker.json"
+    Given I obtain test data file "routing/web-server-rc.yaml"
```
Comment (lihongan) [inline features/networking/egress-ip.feature:357]: `web-server-rc.yaml` creates pod and svc as well, seems here you don't need the service, if that's true then just use `web-server-1.yaml`.

Comment (zhaozhanqi) [reply features/networking/egress-ip.feature]: correct.  updated, thanks

**Comment by lihongan** [comment]
/lgtm
thank you @zhaozhanqi

**Pull request merged**